src/uu/wc/src/wc.rs

```diff
@@ -98,9 +98,8 @@ impl Settings {
     }
 }
 
-static ABOUT: &str = "Display newline, word, and byte counts for each FILE, and a total line if
-more than one FILE is specified. With no FILE, or when FILE is -, read standard input.";
-const USAGE: &str = "{} [OPTION]... [FILE]...";
+static ABOUT: &str = help_about!("wc.md");
```

Comment:
Doesn't make much of a difference, but let's make them both `const` for consistency.

Reply:
Hi, no problem, I didnt want to tinker with original code. Coming right up.

Reply:
Done